src/objects/elements.cc

```diff
@@ -1240,8 +1247,14 @@ class ElementsAccessorBase : public InternalElementsAccessor {
     }
     initial_list_length += nof_property_keys;
 
-    // Collect the element indices into a new list.
-    DCHECK_LE(initial_list_length, std::numeric_limits<int>::max());
+    initial_list_length = params->pageSize((KeyIterationIndex)initial_list_length);
+    if (initial_list_length <= nof_property_keys) {
+      // No space for indices.
+      return keys;
+    }
+
+      // Collect the element indices into a new list.
+      DCHECK_LE(initial_list_length, std::numeric_limits<int>::max());
```

Comment:
fix indent

Reply:
@bhackett1024: Fixed.

But: I hate my auto-formatter. Its so buggy. Sadly, without it, things would be even worse.
Which formatter are you using? Or are you doing everything manually?